### ISSUE TITLE
provide paypal link or donate link

### ISSUE THREAD

**Issue by leela52452** [navit-gps/navit#887]
i have noticed paypal link on your [website](http://www.navit-project.org/) 

if you dont mind can you please update on README or contributing or wiki

**Comment by pgrandin** [comment]
The existing paypal account ( donate@navit-project.org ) is currently managed by cp15, founder of the project who has unfortunately been inactive on Navit for a couple of years now. I have reached out to him to discuss what our options are ( he hasn't contributed code in a while, but he still provides and maintain some critical parts of the infrastructure including the map processing and map serving ).

We started experimenting with another solution some time ago, opencollective ( https://opencollective.com/navit )
The advantage of opencollective is that everything is transparent : you see how much money comes in, and everybody sees how this money is spent. The inconvenient is that they take a substantial fee.

Maybe we should start promoting opencollective again? 
Typically, funds would go toward purchasing devices to test Navit on them, help finance participation in conferences or contribute to infrastructure costs (like map serving).

**Comment by hoehnp** [comment]
@pgrandin: as far as I remember about opencollective, weren't the transaction fees pretty high and this the reason why you and me said it is not so useful to collect money there?

**Comment by pgrandin** [comment]
@hoehnp that's correct. At that time the opencollective contributors were people like you and me, who were both contributing to opencollective and people who could claim reimbursements for device purchases or conferences like fosdem. in that case it wasn't making much sense to give money to the collective and then submit a claim to get some money back. If our users start to pitch in I think that it can be a different story. What do you think?

**Comment by hoehnp** [comment]
@pgrandin: In my opinion before we discuss about what we promote, I would suggest we first have a discussion about what we actually would like to cover from the donations. In my opinion it would be for example reasonable to cover fix costs, e.g. servers, hosting, etc.

**Comment by jkoan** [comment]
What about https://help.github.com/en/articles/displaying-a-sponsor-button-in-your-repository as well?
